### PR TITLE
Avatar and badge

### PR DESCRIPTION
**Pull request by aniketxparihar**
Hey Guys I've made some changes in Avatar and added the Badge Component here. Please Review

### CODE REVIEW

File: Components/Avatar/avatar.html [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+      console.log(open);
+      if (open == false) {
+        sidebar.style.left = "0rem";
+      }
+      if (open == true) {
+        sidebar.style.left = "-30rem";
+      }
+      open = !open;
+    });
+  </script>
```
Comment (samflab) [inline Components/Avatar/avatar.html:175]: Put your JS code in a separate file.

**Review by samflab — APPROVED** [review]

**Comment by samflab** [comment]
Host your project on Netlify so that it becomes easier to review.

File: Components/Avatar/avatar.html (hunk 1 above)
Comment (samflab) [inline Components/Avatar/avatar.html]: Add JS code to a separate file

File: Components/Badge/badge.html [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+      if (open == false) {
+        sidebar.style.left = "0rem";
+
+      }
+      if (open == true) {
+        sidebar.style.left = "-30rem";
+
+      }
+      open = !open;
+    })
```
Comment (samflab) [inline Components/Badge/badge.html]: this as well

**Review by samflab — CHANGES_REQUESTED** [review]

File: Components/Avatar/avatar.html (hunk 1 above)
Comment (aniketxparihar) [reply Components/Avatar/avatar.html]: Sure will add that

**Review by NtshVrm — APPROVED** [review]

**Pull request merged**